pyproject.toml

```diff
@@ -19,7 +19,10 @@ dynamic = ["version"]
 license = { file = "LICENSE" }
 readme = { file = "README.md", content-type = "text/markdown" }
 authors = [{ name = "Equinor" }]
-classifiers = ["Programming Language :: Python"]
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
```

Comment:
Why not "Apache 2.0"?

Reply:
Think it is implied that it is "2.0" and there is only one alternative on the [list of possible classifiers](https://pypi.org/classifiers/)